Toolbar/Internal/GUI/ConfirmDialog.cs

```diff
@@ -36,8 +37,12 @@ internal class ConfirmDialog : AbstractWindow {
 		private Action onCancel;
 		private string okText;
 		private string cancelText;
-
-		internal ConfirmDialog(string title, string text, Action onOk, Action onCancel, string okText = "OK", string cancelText = "Cancel") : base() {
+		// dictionary.cfg
+		// #TOOLBAR_UI_OK = "OK"
+        // #TOOLBAR_UI_CANCEL = "Cancel"
+        //
+        // eg : Localizer.Format("#ID")
+		internal ConfirmDialog(string title, string text, Action onOk, Action onCancel, string okText = Localizer.Format("#TOOLBAR_UI_OK"), string cancelText = Localizer.Format("#TOOLBAR_UI_CANCEL")) : base() {
```

Comment:
Bad syntax, you can't have a function call in the method definition.

Reply:
Yes, I should use the form of parameters,I will edit it and submit it later!

````
private string locStringOk = Localizer.Format("#TOOLBAR_UI_OK");
private string locStringCancel = Localizer.Format("#TOOLBAR_UI_CANCEL");
internal ConfirmDialog(string title, string text, Action onOk, Action onCancel, string okText = locStringOk, string cancelText = locStringCancel) : base() {
			Rect = new Rect(300, 300, Screen.width / 4, 0);
			Title = title;
			Dialog = true;
			Modal = true;

			this.text = text;
			this.onOk = onOk;
			this.onCancel = onCancel;
			this.okText = okText;
			this.cancelText = cancelText;
		}
````